config.js

```diff
@@ -13,6 +13,10 @@ module.exports = {
       label: 'Courses',
       path: '/courses',
     },
+    {
+      label: 'Tech Talks',
+      path: '/tech-talks',
```

Comment:
we should either use camelCase here, or convert /howTo to kebab case

Reply:
Other paths use dash-case (idk what to call it), so I'll convert `howTo` to `how-to`